src/components/TextCard/TextCard.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import cx from 'classnames';
+
+import FileDownload from 'FileDownload';
+import FormattedDate from 'FormattedDate';
+
+import { parseHtml } from 'utils/parse-html';
+import RichText from 'RichText';
+
+type TextCardProps = {
+  children?: React.ReactNode;
+  className?: string;
+  date?: string;
+  dateUpdated?: string;
+  description?: string;
+  fileSize?: string;
+  fileType?: string;
+  hasMetaLabel?: boolean;
+  href: string;
+  id?: string;
+  metaLabel?: string;
+  title: string;
+  type?: 'content' | 'file' | 'taxonomy_term';
+};
+
+export const TextCard = ({
+  children,
+  className,
+  date,
+  dateUpdated,
+  description,
+  fileSize,
+  fileType,
+  hasMetaLabel,
+  href,
+  id,
+  metaLabel,
+  title,
+  type
+}: TextCardProps) => {
+  const classNames = cx('cc-text-card', {
+    [`${className}`]: className
+  });
+
+  return (
+    <article className={classNames} id={id}>
+      {(metaLabel || date) && (
+        <div className="cc-text-card__meta">
+          {/* `metaLabel` temporarily removed as types for search results are not correctly displayed
+          Type label and date logic for cards has been moved to the app side
+          therefore `itemType` prop has been permanently removed
+          TODO: To be revisited for both search results AND text listings
+          https://github.com/wellcometrust/corporate/issues/6730 */}
+          {/* {metaLabel && !['taxonomy_term'].includes(itemType) && (
+            <span className="cc-text-card__meta-label">
+              {metaLabel}
+              &nbsp;
+            </span>
+          )} */}
+          {/* hasMetaLabel is a flag to decide if we want to show a type */}
+          {hasMetaLabel && (
+            <span className="cc-text-card__meta-label">
+              {metaLabel}
+              &nbsp;
+            </span>
```

Comment:
Is the boolean prop necessary? Can we set the condition that if `metaLabel` is not falsy then we display it?

Reply:
It is used in a couple of places so I've erred on the side of caution and kept it in

Reply:
I don't think it's quite as simple as if metaLabel is present or not as there are clearly instances where it is meant to be displayed and others that aren't